src/sentry/receivers/features.py

```diff
@@ -146,11 +145,20 @@ def record_issue_assigned(project, group, user, **kwargs):
     )
 
 
-@issue_resolved_in_release.connect(weak=False)
-def record_issue_resolved_in_release(project, group, user, resolution_type, **kwargs):
-    FeatureAdoption.objects.record(
-        organization_id=project.organization_id, feature_slug="resolved_in_release", complete=True
-    )
+@issue_resolved.connect(weak=False)
+def record_issue_resolved(organization_id, project, group, user, resolution_type, **kwargs):
+    if resolution_type in ('in_next_release', 'in_release'):
+        data_resolution_type = 'resolved_in_release'
```

Comment:
this seems wrong? analytics.events should capture type `in_release`

Reply:
You're half right. This should be `resolution_type` to capture both